perf/micro/immediate-scheduler/observable/from-array.js

```diff
@@ -8,7 +8,7 @@ module.exports = function fromArray(suite) {
     }
 
     var oldFromArrayWithImmediateScheduler = RxOld.Observable.fromArray(args, RxOld.Scheduler.immediate);
-    var newFromArrayWithImmediateScheduler = RxNew.Observable.fromArray(args);
+    var newFromArrayWithImmediateScheduler = RxNew.Observable.from(args);
```

Comment:
More interestingly this morning gave another try on different machine, results was different saying new implementation is faster. Not sure if slower result came out by some of my system configuration. Anyway, will try couple of more run later.